lib/ansible/modules/packaging/os/swupd.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+
+# (c) 2017, Alberto Murillo <alberto.murillo.silva@intel.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from ansible.module_utils.basic import AnsibleModule
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: swupd
+short_description: Manages bundles with M(swupd).
+description:
+  - "Manages bundles with the M(swupd) update manager, which is used by the
+    Clear Linux Project for Intel Architecture."
+version_added: "2.3"
+author: Alberto Murillo (@albertomurillo)
+options:
+  name:
+    description:
+      - Name of the (C)bundle to install or remove.
```

Comment:
bundle is an alias of name so it is an option too. Actually I was thinking on setting the option to bundle and the alias to name since ClearLinux's swupd doesn't manage packages, just bundles.